SVGImageView.h

```diff
@@ -9,16 +9,16 @@
 
 #import "SVGPortability.h"
 
-NS_ASSUME_NONNULL_BEGIN
-
-/// A view that parses, renders, and display an SVG file.
+/// A view that renders an SVG file.
 
 IB_DESIGNABLE
+
 @interface SVGImageView : PSVGView
 
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
I thought that it would make it clear that all variables within it are non-null, but [best practices seem to be otherwise](https://developer.apple.com/swift/blog/?id=25). So I fixed that: https://github.com/arielelkin/PocketSVG/pull/47/commits/861621e80e3d9c6dd68663ffe0f297bc22ea4d0e